Automattic/jetpack
# Protect: Add FlexLayout for AdminSectionHero

Fixes #
Improves responsiveness of the header 

## Proposed changes:

* Adjust the original SeventyFiveLayout component to use flex

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1565

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch load Protect in JT
* Review the various places the FlexLayout component is present
   * HomePage
   * ScanPage (idle, scanning, error states)
   * FirewallPage
   * FirewallFooter
* Double check the header components respond correctly in all viewports
* Ensure no regressions are introduced
